packages/moonstone/Popup/Popup.js

```diff
@@ -276,35 +277,33 @@ class Popup extends React.Component {
 		onClose: PropTypes.func,
 
 		/**
-		 * A function to be run when popup hides. When animating it runs after transition for
-		 * hiding is finished.
+		 * Hide callback function
 		 *
 		 * @type {Function}
 		 * @public
 		 */
 		onHide: PropTypes.func,
 
 		/**
-		 * A function to be run when a key-down action is invoked by the user.
+		 * A key-down event callback function
 		 *
 		 * @type {Function}
 		 * @public
 		 */
 		onKeyDown: PropTypes.func,
 
 		/**
-		 * A function to run when popup shows. When animating, it runs after transition for
-		 * showing is finished.
+		 * Called when popup shows.
 		 *
-		 * Note: The function does not run if Popup is initially opened and non animating.
+		 * Note: The function does not run if Popup is initially opened and not animating.
 		 *
 		 * @type {Function}
 		 * @public
 		 */
 		onShow: PropTypes.func,
 
 		/**
-		 * When `true`, the popup is rendered. Popups are rendered into the
+		 * Indicates the popups will be rendered into the
```

Comment:
Perhaps:  Controls the visibility of the Popup.

I'm not sure whether it's relevant to mention the floating layer here, as we do in the description